toolkit/tools/unravel/unravel.go

```diff
@@ -69,6 +69,8 @@ func main() {
 		}
 
 		if *runCheck == "y" {
+			logger.Log.Warn("Running package tests (make argument 'RUN_CHECK' set to 'y'). Will add the 'ca-certificates' package and enable networking for package builds.")
```

Comment:
```suggestion
			logger.Log.Warn("Running package tests. Will add the 'ca-certificates' package and enable networking for package builds.")
```
Do not reference specific Makefile options inside the tool. The unravel tool should be thought of as a standalone binary that can be run independently of our Makefile.

Reply:
You're right. What do you think about the change?